shell/server/backup.js

```diff
@@ -24,10 +24,18 @@ const GrainInfo = Capnp.importSystem("sandstorm/grain.capnp").GrainInfo;
 const TOKEN_CLEANUP_MINUTES = 15;
 const TOKEN_CLEANUP_TIMER = TOKEN_CLEANUP_MINUTES * 60 * 1000;
 
+const uploadTokens = {};
```

Comment:
Why switch to an in-memory map? While technically this probably works with Blackrock due to the use of sticky sessions, I would rather not make such assumptions if we can avoid it.